PixiEditor/ViewModels/SubViewModels/Main/ToolsViewModel.cs

```diff
@@ -41,12 +42,17 @@ public ToolsViewModel(ViewModelMain owner)
                 new CircleTool(), new RectangleTool(), new EraserTool(), new ColorPickerTool(), new BrightnessTool(),
                 new ZoomTool()
             };
-            SetActiveTool(ToolType.Move);
+            SetActiveTool(typeof(MoveTool));
         }
 
-        public void SetActiveTool(ToolType tool)
+        public void SetActiveTool(Type toolType)
```

Comment:
This parameter shouldn't be `Type`, instead, it should be a `Tool`, because you don't want anyone to put any type that exists, just a type of tool